### PR TITLE
Implement get tasks by plan and owner

### PR DESCRIPTION
**Pull request by ellykits**

### CODE REVIEW

File: src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java [hunk 1]
```diff
@@ -228,4 +228,28 @@ public void testFindAllIdsShouldLimitByGivenParam() {
 		assertEquals(1542027762554l, idsModel.getRight().longValue());
 	}
 
+	public void testGetTasksByOwnerAndPlan() {
```
Comment (githengi) [inline src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java]: Add a `@Test` annotation

File: src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java [hunk 2]
```diff
@@ -183,7 +183,7 @@ public void testGetTasksByCampaignAndGroup() {
 		task.setServerVersion(System.currentTimeMillis());
 		taskRepository.add(task);
 
-		assertEquals(2, taskRepository.getTasksByPlanAndGroup("IRS_2018_S1", "2018_IRS", 0));
+		assertEquals(2, taskRepository.getTasksByPlanAndGroup("IRS_2018_S1", "2018_IRS", 0).size());
```
Comment (githengi) [inline src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java]: Add a `@Test` annotation to this method

File: src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java [hunk 3]
```diff
@@ -228,4 +228,28 @@ public void testFindAllIdsShouldLimitByGivenParam() {
...
+		Task task = new Task();
+		task.setIdentifier("tsk-2332-j");
+		task.setPlanIdentifier("IRS_2018_S");
+		task.setGroupIdentifier("2018_IRS-3734");
+		task.setBusinessStatus("Not Visited");
+		task.setStatus(TaskStatus.DRAFT);
+		task.setServerVersion(System.currentTimeMillis());
+		taskRepository.add(task);
+
+		assertEquals(2, taskRepository.getTasksByPlanAndOwner("IRS_2018_S1", "demouser", 0).size());
```
Comment (githengi) [inline src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java]: Why does the tasks increase to 2 considering the task created above does not have an `owner` so should not be returned.

Here we expect a list of size 1.

Set the owner then update the task and then assert that list returned has 2

File: src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java (hunk 2 above)
Comment (githengi) [inline src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java]: Add a @Test annotation to the method

File: src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java (hunk 3 above)
Comment (ellykits) [reply src/test/java/org/opensrp/repository/postgres/TaskRepositoryTest.java]: Cool

**Pull request merged**